data/eventStorage.json

```diff
@@ -0,0 +1,141 @@
+{
+  "events" : [ {
+    "eventName" : "fixing computers",
+    "roles" : [ {
+      "roleName" : "mechanic",
+      "currentQuantity" : "0",
+      "requiredQuantity" : "1"
+    }, {
+      "roleName" : "engineer",
+      "currentQuantity" : "0",
+      "requiredQuantity" : "4"
+    } ],
+    "startDate" : "10/1/2023 1600",
+    "endDate" : "14/5/2023 1500",
+    "location" : "home",
+    "description" : "fix computers",
+    "materials" : [ {
+      "material" : "computers",
+      "currentQuantity" : "0",
+      "requiredQuantity" : "50"
+    } ],
+    "budget" : "120.00",
+    "assignedVolunteers" : [ ],
+    "maxVolunteerSize" : "9223372036854775807"
```

Comment:
qiguai number but ok

Reply:
this is the maximum long value. eshow will display as No limit

Reply:
so no need to fix at all

Reply:
too bad cs2100 only teach about the 2147483647

Reply:
hao I change it cos idk whats 9223372036854775807 either lol

Reply:
changed it back for some events